romancal/regtest/test_dark_current.py

```diff
@@ -0,0 +1,82 @@
+"""Regression test for the Dark current subtraction step."""
+import pytest
+
+from romancal.stpipe import RomanStep
+from .regtestdata import compare_asdf
+
+
+@pytest.mark.bigdata
+def test_dark_current_subtraction_step(rtdata, ignore_asdf_paths):
+    """ Function to run and compare Dark Current subtraction files. Note: This
+        should include tests for overrides etc. """
+    rtdata.get_data(
+        "WFI/image/r0019101001023012008_01101_0173_WFI01_jumpstep.asdf")
+    rtdata.input = "r0019101001023012008_01101_0173_WFI01_jumpstep.asdf"
```

Comment:
Isn't dark run before jump? Shouldn't the input be from a step before dark?

Reply:
Good catch, I was also using an older input file from INS and I've updated to the latest.